src/main/java/uk/gov/hmcts/ccd/domain/service/cauroles/AuthorisedCaseAssignedUserRolesOperation.java

```diff
@@ -23,6 +23,12 @@ public AuthorisedCaseAssignedUserRolesOperation(final @Qualifier("default") Case
         this.cauRoleValidator = cauRoleValidator;
     }
 
+    public void addCaseUserRoles(List<CaseAssignedUserRole> caseUserRoles) {
+        // NB: Although there are no user based authorisation steps performed here ...
```

Comment:
You made me find out `https://www.quora.com/What-does-the-abbreviation-N-B-mean-and-why-do-people-use-it`. :)